nixos/doc/manual/configuration/xfce.xml

```diff
@@ -87,15 +87,15 @@
             (look at journalctl --user -b).
```

Comment:
While you're here, can you wrap `journalctl --user -b` in `<command>` / `</command>`?